sycl/include/sycl/ext/xilinx/xrt.hpp

```diff
@@ -0,0 +1,127 @@
+//==--------- xrt.hpp - SYCL XRT backend ---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl/backend.hpp>
+#include <CL/sycl/program.hpp>
+#include <CL/sycl/detail/pi.h>
+#include <CL/sycl/backend.hpp>
+
+#include <xrt/xrt_kernel.h>
+#include <xrt.h>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace detail {
```

Comment:
Do not tell me you want C++14 :-)
```suggestion
namespace sycl::detail {
```

Reply:
Ah yes, I was wrong here on the reason. You really 2 scopes.